app/assets/stylesheets/components/page_specific/_stripe_banner.scss

```diff
@@ -0,0 +1,70 @@
+// Stripe banner which appears on the homepage
+//
+// Styleguide Stripe banner
+
+.stripe-banner {
+  background: #f3f3f3 url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI4IiBoZWlnaHQ9IjgiIHZpZXdCb3g9IjAgMCA4MCA4MCIgZW5hYmxlLWJhY2tncm91bmQ9Im5ldyAwIDAgODAgODAiPjxwYXRoIHN0cm9rZT0iIzAwMCIgZD0iTTAgODBMODAgMCIvPjwvc3ZnPg==);
```

Comment:
base64-ing svg increases file size: http://calendar.perfplanet.com/2014/tips-for-optimising-svg-delivery-for-the-web/ (see section 4.)